test/androidTest/java/org/thoughtcrime/securesms/RepliesEspressoTest.java

```diff
@@ -0,0 +1,82 @@
+package org.thoughtcrime.securesms;
+
+import android.support.test.filters.LargeTest;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.thoughtcrime.securesms.espresso.Helper;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class RepliesEspressoTest {
+    @Rule
+    public ActivityTestRule<ConversationListActivity> mainActivityRule =
+            new ActivityTestRule(ConversationListActivity.class, true, false);
+
+    @Test
+    public void buttonExists(){
+        Helper helper = new Helper(mainActivityRule);
+        String testSend = helper.randString();
```

Comment:
The two last tests have a space between the helper and the test string(s). For consistency, I think they should all be the same (an I prefer having the space)